Sources/SwiftDocC/SwiftDocCMarkdownOutput/MarkdownOutputNode.swift

```diff
@@ -149,35 +153,36 @@ extension MarkdownOutputNode {
 
 // MARK: I/O
 extension MarkdownOutputNode {
-    /// Data for this node to be rendered to disk
-    public var data: Data {
-        get throws {
-            let encoder = JSONEncoder()
-            encoder.outputFormatting = [.prettyPrinted, .sortedKeys, .withoutEscapingSlashes]
-            let metadata = try encoder.encode(metadata)
-            var data = Data()
-            data.append(contentsOf: Self.commentOpen)
-            data.append(metadata)
-            data.append(contentsOf: Self.commentClose)
-            data.append(contentsOf: markdown.utf8)
-            return data
-        }
+    /// Data for this node to be rendered to disk as a markdown file. This method renders the metadata as a JSON header wrapped in an HTML comment block, then includes the document content.
+    public func generateDataRepresentation() throws -> Data {
```

Comment:
Question: is the client meant to do anything _else_ with this raw data or is it only "be rendered to disk as a markdown file"? 

Alternatively, is the _client_ meant to write these files at all? I was sort of under the impression that it was DocC that created these files and the the _client_ would only read them. If the client doesn't create and write its own files, does this need to be SPI or could it be `package` or `internal` access?